lavamoat/browserify/beta/policy.json

```diff
@@ -656,6 +656,8 @@
         "Notification.maxActions": true,
         "Notification.permission": true,
         "Notification.requestPermission": true,
+        "PushManager.getSubscription": true,
+        "PushManager.subscribe": true,
```

Comment:
New Policy is from `policy-override` file as the `yarn lavamoat:auto` script is not adding this property.